hack/cloudbuild.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+
+# with nounset, these will fail if necessary vars are missing
+echo "GIT_TAG: ${GIT_TAG}"
+echo "PULL_BASE_REF: ${PULL_BASE_REF}"
+echo "PLATFORM: ${PLATFORM}"
+
+# debug the rest of the script in case of image/CI build issues
+set -o xtrace
+
+REPO="gcr.io/k8s-staging-sig-storage"
+
+CONTROLLER_IMAGE="${REPO}/objectstorage-controller"
+SIDECAR_IMAGE="${REPO}/objectstorage-sidecar"
+
+# args to 'make build'
+export DOCKER="/buildx-entrypoint" # available in gcr.io/k8s-testimages/gcb-docker-gcloud image
+export BUILD_ARGS="--push"
+export PLATFORM
+export SIDECAR_TAG="${SIDECAR_IMAGE}:${GIT_TAG}"
+export CONTROLLER_TAG="${CONTROLLER_IMAGE}:${GIT_TAG}"
+
+# build in parallel
+make --jobs --output-sync build
```

Comment:
Is there any benefit for multiprocessing here?

Reply:
In my local testing, this increases the build speed, but it isn't significant. Maybe a 20-30% improvement. I am also testing on macos where the linux native tools don't run as efficiently. It would be hard to guess whether the improvement on the linux build environments would be more or less impactful.